assist/README.md

```diff
@@ -1,8 +1,141 @@
 RACK ASSIST
 ===========
 
+# Introducing RACK ASSIST
+
 (Automated Software Structure Ingestion and System Testing)
 
+The ASSIST toolkit contains several tools that can be used with RACK.
+
+ * ASSIST-DC  -- Data Collection
+ * ASSIST-DI  -- Data Ingestion
+ * ASSIST-DV  -- Data Verification
```

Comment:
I'm thinking that if we keep using the acronyms `DC`, `DI`, `DV`, it may always take me some brain power to remember what the letters stood for.

What do you think of an alternative like:

- ASSIST-Collect(ion?)
- ASSIST-Ingest(ion?)
- ASSIST-Verif(y/ication?)

Not sure I love it much better, but something to consider.

Reply:
I debated that myself.  In the end, I stuck with the two-letter acronym because it's more distinctive and so it helps clarify you are talking about something specific as opposed to a more general reference, especially in verbal communications.

I'm open to debate on this though.